include/openthread/platform/radio.h

```diff
@@ -866,6 +880,31 @@ bool otPlatRadioIsCoexEnabled(otInstance *aInstance);
  */
 otError otPlatRadioGetCoexMetrics(otInstance *aInstance, otRadioCoexMetrics *aCoexMetrics);
 
+/**
+ * Enable or disable CSL receiver.
+ *
+ * @param[in]  aInstance     The OpenThread instance structure.
+ * @param[in]  aCslPeriod    CSL period, 0 for disabling CSL.
+ * @param[in]  aExtAddr      The extended source address of CSL receiver's parent device(requested by NRF52840).
```

Comment:
I'm not sure about this. As I remember, in my previous implementation for CSL on Nordic, it would be better to not split this process. I think we can reconsider this when I try to push the PR for CSL on Nordic. Thoughts?

Reply:
fine to me.

Reply:
Rather than saying "requested by NRF52840", can we put a reason why it is useful for the platform driver to know the parent's extended address in this case?